ui/src/views/404/index.jsx

```diff
@@ -1,5 +1,52 @@
 // @flow
+import { colour } from 'pasteup';
+import Logo from 'assets/images/guardian-logo-320.svg';
+import Navigation from './Navigation';
+import Footer from './Footer';
 
-import style from './style.js.scss';
+import {
+    heading,
+    fluidWrap,
+    topbar,
+    subHeadingContainer,
+    subHeading,
+    para,
+} from './style.js.scss';
 
-export default () => <div style={style.message}>404</div>;
+export default () =>
+    <div style={fluidWrap}>
+        <div style={topbar}>
+            <a
+                href="/"
+                style={{
+                    color: '#005689',
+                    textDecoration: 'none',
+                    fontFamily: 'arial',
+                    padding: '8px 10px 7px',
+                    display: 'block',
+                }}>
+                Home
+            </a>
+        </div>
+        <Logo
```

Comment:
To elaborate on this this example:
> As an example, I think its good to know you are using the logo as an svg vs using the logo as an svg with a fallback

I create a component with two icons. One is the logo, the only is something else less important like an icon of the world next to a label. With the former I know that if I am using an SVG I want to make sure there is a fallback. The latter I don't want a fallback because the label does enough of the job anyway and the extra bytes to send to every client for a fallback isn't worth it. 

As the developer building this component I would argue I would have the best idea of what is critical and what is secondary, as it won't always be the same per subcomponent (smaller component within the component I am building such as an icon)